presto-main/src/main/java/com/facebook/presto/metadata/DiscoveryNodeManager.java

```diff
@@ -189,15 +188,13 @@ public synchronized Node getCurrentNode()
 
     private static URI getHttpUri(ServiceDescriptor descriptor)
     {
-        // favor https over http
```

Comment:
@martint, updated the PR.
FYI, this one depends on https://github.com/airlift/airlift/pull/291.